src/ccnl-riot/src/ccn-lite-riot.c

```diff
@@ -547,6 +547,9 @@ ccnl_send_interest(struct ccnl_prefix_s *prefix, unsigned char *buf, int buf_len
     int ret = -1;
     int len = 0;
     ccnl_interest_opts_u default_opts;
+    default_opts.ndntlv.nonce = 0;
+    default_opts.ndntlv.mustbefresh = false;
+    default_opts.ndntlv.interestlifetime = CCNL_INTEREST_TIMEOUT;
```

Comment:
Have you checked if the value is set later on in milliseconds? The define points to a constant which is in seconds, but the specification expects the value to be in milliseconds.

Reply:
No, I haven't. I just checked that this value is asigned in `src/ccnl-pkt/src/ccnl-pkt-ndntlv.c` in `ccnl_ndntlv_bytes2pkt()` as a default value.

Reply:
I've opened issue #210 which addresses this issue. Do you think it makes sense to change
``` C
 default_opts.ndntlv.interestlifetime = CCNL_INTEREST_TIMEOUT;
```
to 
``` C
 default_opts.ndntlv.interestlifetime = CCNL_INTEREST_TIMEOUT * 1000;
```
since in ``ccnl-relay.c`` the value is divided by 1000?

Reply:
Ok, in line 570 a nonce will be generated if it was set to zero.